src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,54 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.exception.DukeException;
+
+/**
+ * The Event class extends the DatedTask class and represents an event with a start and end date.
+ */
+public class Event extends DatedTask {
+    private static final long serialVersionUID = 103;
```

Comment:
Since this is a constant, may want to name it all uppercase using underscore to separate words e.g. `SERIAL_UID`